src/views/Dashboard/Customer/Home/RequestTest/MenuCard.vue

```diff
@@ -3,7 +3,9 @@
     v-card.menu-card
       v-row.pa-3
         v-col(cols="4")
-          ui-debio-avatar(:src="icon" size="135" rounded)
+          v-img(v-if="!icon" src="@/assets/debio-logo.png" height="135" width="135" contain)
+
+          ui-debio-avatar(v-else :src="icon" size="135" rounded)
```

Comment:
please add `TODO` here, so in the future, we can improve in a more efficient way. Thank you